src/control/cmd/dmg/storage.go

```diff
@@ -337,3 +338,29 @@ func (cmd *nvmeSetFaultyCmd) Execute(_ []string) error {
 	}
 	return cmd.makeRequest(ctx, req)
 }
+
+// storageReplaceCmd is the struct representing the replace storage subcommand
+type storageReplaceCmd struct {
+	smdQueryCmd
+	OldDevUUID  string `long:"old-dev" description:"Device UUID of hot-removed SSD" required:"1"`
+	NewDevUUID  string `long:"new-dev" description:"Device UUID of new device" required:"1"`
+	NoReint     bool   `long:"no-reint" description:"Bypass reintegration of device and just bring back online."`
+}
+
+// Execute is run when storageReplaceCmd activates
+// Replace a hot-removed device with a newly plugged device, or reuse a FAULTY device
+func (cmd *storageReplaceCmd) Execute(_ []string) error {
+	if cmd.OldDevUUID == cmd.NewDevUUID {
+		cmd.log.Info("WARNING: Attempting to reuse a previously set FAULTY device!")
+	}
+
+	// TODO: Implement no-reint flag option
```

Comment:
I think this should be implemented in another patch. I will remove the TODO for now

Reply:
Hmm, all I meant was that you already have the NoReint bool exposed via the dmg command surface, so you could just add that value to the request. It would still be a no-op on the server side, but at least when it's implemented you would only have to update the stuff in the server.

That having been said, if this is going to take a while to be implemented, maybe we shouldn't expose it at all until then, otherwise it could be confusing for users.

Reply:
Ok makes sense, I will add the noreint bool to the proto req structs, and I will have an output to user if the flag is set to notify that the option is not currently implemented. I would like to keep the TODOs in here if possible, as it will be implemented sooner rather than later.